charts/csm-application-mobility/Chart.yaml

```diff
@@ -6,13 +6,13 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.3.0
```

Comment:
These changes have to be merged into the release branch otherwise it will kick off the release workflow.

Reply:
@shaynafinocchiaro advise, please, which branch should be used. there are 3 branches mentioning 1.11

Reply:
done

Reply:
Sorry, I guess that branch was created by mistake and will be deleted soon, this is the true one [release-v1.11.0](https://github.com/dell/helm-charts/tree/release-v1.11.0)